set-web/src/main/java/eu/europeana/set/web/service/impl/BaseUserSetServiceImpl.java

```diff
@@ -775,6 +778,13 @@ void validateEntityBestItemsSet(UserSet webUserSet)
           UserSetI18nConstants.USERSET_VALIDATION_ENTITY_REFERENCE,
           new String[] {WebUserSetModelFields.SUBJECT, String.valueOf(subject)});
     }
+    // if present check of entity uri pattern
+    if (StringUtils.startsWith(subject.get(0), WebUserSetFields.DATA_EUROPEANA_BASE_URL) && StringUtils.contains(subject.get(0), WebUserSetFields.ENTITY_URI_BASE)) {
```

Comment:
here we perform the URI validation, we don't accept URLs with base anymore, we have to respond with 400